src/detect-engine-mpm.c

```diff
@@ -643,6 +643,13 @@ uint16_t PatternMatchDefaultMatcher(void)
             }
         }
 
+#ifndef BUILD_HYPERSCAN
+    if (strcmp(mpm_algo, "hs") == 0) {
+        FatalError(SC_ERR_INVALID_VALUE, "Hyperscan (hs) support for mpm-algo is "
+                   "not compiled into Suricata. Add --with-libhs-includes and "
+                   "--with-libhs-libraries to configure.");
```

Comment:
I can live with this error. But I find in most cases if Hyperscan is simply installed, Suricata will pick it up without these options.